content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -224,7 +224,7 @@ Some points to note:
 
 - `usages` has to be '`client auth`'
 - `request` is the base64 encoded value of the CSR file content.
-  You can get the content using this command: ```cat myuser.csr | base64 | tr -d "\n"```
+  You can get the content using this command: ```base64 -w 0 myuser.csr```
```

Comment:
This `-w 0` flag doesn't work on macOS.

Reply:
Ah! Humbly, I didn't consider that base64 could have also a different implementation on other UNIX variants.
Oddly enough, on macOS the default isn't to wrap lines. The GNU coreutils version has a default value of '-w 76'.
OK. I guess this can be closed?

Reply:
I tend to close this PR because the current command is more general.
If you really want to remind users that `-w 0` can be used on *nix systems, you can revise the PR to include that precondition.

Reply:
No. I think we can close it. Thanks.